src/schema/me/create_credit_card_mutation.js

```diff
@@ -0,0 +1,57 @@
+import { GraphQLInt, GraphQLObjectType, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { GravityIDFields } from "schema/object_identification"
+
+export const CreditCardType = new GraphQLObjectType({
+  name: "CreditCard",
+  fields: () => ({
+    ...GravityIDFields,
+    brand: {
```

Comment:
can some of these fields be further strengthened with `new GraphQLNonNull(GraphQLString)`, etc.?

Reply:
None of these are required fields in gravity, so I don't know if we want to be strict here 😬. I can add it on the `token` input though since that is required.